ISISComputingGroup/ibex_gui
# Ticket1171 indicating disconnected blocks & Ticket1157 display alarm states of blocks

https://github.com/ISISComputingGroup/IBEX/issues/1171
Includes https://github.com/ISISComputingGroup/IBEX/issues/1157 too as they have shared code.

Block states are reflected by coloured borders around the value field. Colours are the conventional purple for disconnected, orange for minor alarm, red for major alarm. "Disconnected" overrides all other states including runcontrol states.

To test:
Disconnected state: Set any block to disconnected by giving it a nonsense PV address
Alarm: move a motor that's observed by a block to its limit - this should trigger a minor alarm and  turn the blocks border orange
.
You can confirm alarm states of blocks by opening the console and getting the alarm severity with CAGET [pv_address].SEVR
